src/Symfony/Component/Mailer/Transport/FileTransport.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mailer\Transport;
+
+use Psr\Log\LoggerInterface;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\Mailer\SentMessage;
+
+/**
+ * Pretends messages have been sent, but just ignores them.
+ *
+ * @author Fabien Potencier <fabien@symfony.com>
+ *
+ * @experimental in 4.3
```

Comment:
4.4?

Reply:
Hum, it depends if the Mailer component will still be experimental in 4.4 or not.
I will remove this line if it's not the case. :+1:

Reply:
experimental annotations was removed yesterday

Reply:
Thanks for the info, I will remove it :+1: